addons/http_routing/models/ir_http.py

```diff
@@ -349,7 +349,7 @@ def get_nearest_lang(cls, lang_code):
             return lang_code
 
         short = lang_code.partition('_')[0]
-        return next((code for code in lang_codes if code.startswith(short)), None)
+        return short and next((code for code in lang_codes if code.startswith(short)), None)
```

Comment:
So it returns the same value as `if not lang_code: return None`
```suggestion
        if not short:
           return None
        return next((code for code in lang_codes if code.startswith(short)), None)
```

Reply:
Already return the same value `None`with my fix:
![image](https://github.com/user-attachments/assets/9b652617-0722-447d-9985-c36a99ccb3ec) :thinking: 

but we could make it more explicit